OmarEspinosaCastro/30-Agosto/List.java

```diff
@@ -0,0 +1,29 @@
+
+package collection;
+
+public class List {
+    public static void main(String[] args) {
+        Scanner in =new Scanner(System.in);
+        int numberElements = in.nextInt();
+        ArrayList<Integer> lista = new ArrayList();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices